test/runtests.jl

```diff
@@ -1,8 +1,10 @@
-using Aqua, CpuId, ReTestItems, ReactiveMP
+using Aqua, CpuId, ReTestItems, ReactiveMP, Hwloc
```

Comment:
Do we need CpuId then? Can we just use `Hwloc` in all situations?

Reply:
I think the only argument for `CpuId` is that it is very fast since you only have to run one instruction `cpuid`.
We only need it once so the theoretical performance increase is non-existant or minimal at best 😄.

Should I just remove the CpuId requirement 👍 👎 ?